card/mod/05_standard/set/all/rich_html/form.rb

```diff
@@ -182,7 +182,7 @@ def content_field form, options={}
 # FIELD VIEWS
 
   view :editor do |args|
-    text_area :content, :rows=>3, :class=>'tinymce-textarea card-content', :id=>unique_id, "data-card-type-code"=>card.type_code
+    text_area :content, :rows=>3, :class=>'tinymce-textarea card-content', :id=>unique_id
   end
```

Comment:
nevermind; we never want tinymce and ace at the same time!

would be nice if the ace editor stuff only showed up if BOTH the ace-editor class AND the type-code were there.